kobo/settings.py

```diff
@@ -601,3 +643,6 @@ def __init__(self, *args, **kwargs):
 MONGO_CONNECTION = MongoClient(
     MONGO_CONNECTION_URL, j=True, tz_aware=True, connect=False)
 MONGO_DB = MONGO_CONNECTION[MONGO_DATABASE['NAME']]
+
+# Number of tries Celery can do to send data to external endpoint
+HOOK_MAX_RETRIES = 3
```

Comment:
Could this be moved into `CONSTANCE_CONFIG`? That way, superusers could set it by going to `https://theirsite/admin`.

Reply:
Done.
https://github.com/kobotoolbox/kpi/commit/40640497f8f8b368050c4d145f8dda11eb24fb0c